lib/sogou/search/api/version.rb

```diff
@@ -1,7 +1,7 @@
 module Sogou
   module Search
     module Api
-      VERSION = '1.0.0'
+      VERSION = '1.0.1'
```

Comment:
I think this should be `1.1.0`, this adds a bit more changes than simple bugfix/patch.